runtime-modules/service-discovery/Cargo.toml

```diff
@@ -56,15 +61,52 @@ rev = 'c37bb08535c49a12320af7facfd555ce05cce2e8'
 default-features = false
 features = ['derive']
 package = 'parity-scale-codec'
-version = '1.0.0'
+version = '1.1.0'
```

Comment:
is this required specifically or is it just a result of doing cargo-update ?

Reply:
I'm not sure how it happened. Whether it was changed somehow automatically or it was my mistake. Anyway, I downgrade it back to '1.0.0'.